CODEOWNERS

```diff
@@ -0,0 +1,82 @@
+# Catch-all
+* @tgraf @manalibhutiyani
+
+# BPF Data Path
+bpf/* @joestringer @borkmann @jrfastab @manalibhutiyani
+pkg/maps/* @joestringer @borkmann @jrfastab @manalibhutiyani
+pkg/bpf/* @joestringer @borkmann @jrfastab @manalibhutiyani
+# TODO Where are "KProxy kernel" and "XDP Prefilter" parts?
```

Comment:
Most parts are in `bpf/` for XDP and from golang side the main bits are in `pkg/maps/cidrmap/cidrmap.go`, `pkg/policy/prefilter.go`, `daemon/prefilter.go`.